config/config.txt

```diff
@@ -599,3 +599,6 @@ CACHE_ASSETS 0
 ## The number of unique PR announcements allowed per round
 ## This makes sure that a single person can only spam 3 reopens and 3 closes before being ignored
 PR_ANNOUNCEMENTS_PER_ROUND 5
+
+## When enabled, grants those with R_DEBUG permissions to use the server profile debug functionality
+ALLOW_ADMIN_PROFILING
```

Comment:
invert this. If the config doesn't exist (ie, existing configs) it should retain old behavior.

changing the default in code means you have to do `allow_admin_profiling 0` to disable it which is lame, changing it to forbid_admin_profiling would be less lame.